src/_sass/home.scss

```diff
@@ -111,6 +111,7 @@
   font-family: $tilt2-font-sans;
   font-size: $tilt2-button-font-size--large;
   font-weight: $tilt2-font-weight-sans-bold;
+  padding: 0;
```

Comment:
fwiw, it should probably be min-width instead of width and white-space: nowrap, so that the button never overflows (regardless of zoom level/font size)

Reply:
Thanks @nicks . Fixed.